src/data/backup.ts

```diff
@@ -13,20 +13,22 @@ export interface BackupContent {
   agent_ids?: string[];
 }
 
-export interface BackupAddon {
-  name: string;
-  slug: string;
-  version: string;
-}
-
-export interface BackupContentExtended extends BackupContent {
+export interface BackupData {
   addons: BackupAddon[];
   database_included: boolean;
   folders: string[];
   homeassistant_version: string;
   homeassistant_included: boolean;
 }
 
+export interface BackupAddon {
+  name: string;
+  slug: string;
+  version: string;
+}
+
+export interface BackupContentExtended extends BackupContent, BackupData {}
```

Comment:
`BackupContentExtended` is just the combination of `BackupContent` and `BackupData`?

So:
```ts
export type BackupContentExtended2 = BackupContent & BackupData;
```

Reply:
Yeah it was using interface so I didn't changed but I also prefer using type